modules/c++/math.linear/include/math/linear/Matrix2D.h

```diff
@@ -1082,6 +1083,21 @@ class Matrix2D
     }
 
     /*!
+     *  Negation operator;
+     *
+     *  \code
+           B = -A;
+     *  \endcode
+     *
+     */
+    Matrix2D operator-() const
+    {
+        Matrix2D mneg(*this);
```

Comment:
Another nit... member variables start with `m`.  No Hungarian notation aside from that.  So in general, avoid abbreviating with just an `m` as the first letter (but something like `max` is fine if it's obviously part of a word).  So here you'd use either `neg`, or if you wanted to denote that it's a matrix, `matNeg` or something like that.